example/tests/test_generic_viewset.py

```diff
@@ -11,6 +11,16 @@ class GenericViewSet(TestBase):
     """
     Test expected responses coming from a Generic ViewSet
     """
+
+    def setUp(self):
+        super(GenericViewSet, self).setUp()
+
+        setattr(settings, 'JSON_API_FORMAT_KEYS', 'dasherize')
```

Comment:
The attributes used in the JSON were dasherized so this is needed. Previously, the tests passed by coincidence due to another test changing the format to 'dasherize'.